.github/workflows/gitleaks.yml

```diff
@@ -1,25 +1,32 @@
-name: gitleaks
 on:
   workflow_call:
+    secrets:
+      GITLEAKS_LICENSE_KEY:
+        required: true
 
 jobs:
-  scan:
-    name: gitleaks
+  gitleaks:
     runs-on: ubuntu-latest
+    permissions:
+      actions: read
+      contents: read
+      security-events: write
     steps:
-      -
-        uses: actions/checkout@v3
+      - name: Checkout
+        uses: actions/checkout@v2
         with:
           fetch-depth: 0
-      -
-        name: Configure GitLeaks
-        if:  ${{ hashFiles('.gitleaks.toml') == '' }}
+      - name: Configure GitLeaks
+        if:  hashFiles('.gitleaks.toml') == ''
         run: |
           curl -O https://raw.githubusercontent.com/smallstep/workflows/master/.gitleaks.toml
-      -
-        uses: gitleaks/gitleaks-action@v2
+      - uses: gitleaks/gitleaks-action@v2
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-          GITLEAKS_LICENSE: ${{ secrets.GITLEAKS_LICENSE_KEY}} # Only required for Organizations, not personal accounts.
-          GITLEAKS_CONFIG: '.gitleaks.toml'
-          GITLEAKS_NOTIFY_USER_LIST: '@dopey,@maraino,@joshdrake'
+          GITLEAKS_LICENSE: ${{ secrets.GITLEAKS_LICENSE_KEY }}
```

Comment:
Do we need a license for open-source repos?

Reply:
Yes. You need a license for every repo, because we're getting charged based on our usage.